tests/console/validate_dud_addon_repos.pm

```diff
@@ -1,6 +1,6 @@
 # SUSE's openQA tests
 #
-# Copyright 2020 SUSE LLC
+# Copyright 2022 SUSE LLC
```

Comment:
I don't think we need to change this file yet, we can assume that it works, there are too many repos to check and anyway the customer is not going to add the updates like that, so this safety we can thing later how to implement it and even how to do it in the fastest way.

Reply:
If we don't change this file, the test will fail. Refer case: https://openqa.suse.de/tests/9371636#step/validate_dud_addon_repos/8.  It will always check the expected repo.

Reply:
this child job we don't need it, we need to use any of the test suite running interactive installation in maintenance where that module is not scheduled.

Reply:
for maintenance we shouldn't have used test suite `dud_development_tools_dev` but any test suite where we can apply this method, schedule should remain the same for product validation (existing test suite in YaST group). Does it answer your question?